### ISSUE TITLE
Cannot pull/push

### ISSUE THREAD

**Issue by wernight** [Andr3as/Codiad-CodeGit#36]
Trying to do a pull/push returns:

    {"status":"error","message":"Failed to pull repo!"}

Environment:

  * Using [wernight/codiad](https://hub.docker.com/r/wernight/codiad/) Docker image
  * Running inside the Terminal plugin (or directly inside the shell as PHP's user):

        $ ssh -T git@gitlab.example.com
        Welcome to GitLab, John Doe!

  * Running from inside the Terminal plugin a `git pull` also works.

PHP's logs seem pretty parse about the cause of this issue. I only see:

> 2016/06/28 22:22:22 [error] 13#0: *469 connect() failed (111: Connection refused) while connecting to upstream, client: ::ffff:10.234.0.2, server: _, request: "GET /plugins/Codiad-CodeGit-master/controller.php?action=status&path=livreslibertyscom HTTP/1.1", upstream: "fastcgi://[::1]:9000", host: "codiad.example.com", referrer: "http://codiad.example.com/"

**Comment by wernight** [comment]
It's visibly due to the reliance on some binaries as this issue is only on Alpine Linux and not on Debian based.

**Comment by Andr3as** [comment]
I'm currently not able to test it with Alpine Linux. Did you checked the [dependencies](https://github.com/Andr3as/Codiad-CodeGit#requirements)?

**Comment by wernight** [comment]
Yes dependencies are met. The simplest to repro this bug is using Docker:

    $ docker run --rm -p 8080:80 wernight/codiad:alpine

**Comment by Andr3as** [comment]
Sorry for my late reply. I tested it today with your docker image and was not able to reproduce your issue.

**Comment by wernight** [comment]
You're sure you're using the `:alpine` tag? I don't remember upgrading it, so it should still reproduce (that is not pull or push without actually giving an error if I recall).

**Comment by Andr3as** [comment]
Okay, thanks, I missed that.

After some digging: Alpine does not provide Bash as shell.
Try to change the following in [`expect.sh`](https://github.com/Andr3as/Codiad-CodeGit/blob/master/scripts/expect.sh)
`#!/bin/bash`
to
`#!/bin/sh`

I was not able to test it. My alpine docker instance tells me always the following error
 ```shell
   $ ssh -T git@github.com
   ssh: Could not resolve hostname github.com: Try again
```

After fixing this, I'll come back to you.

**Comment by wernight** [comment]
It sounds like you cannot access network from within your container. That
shouldn't be specific to Alpine.

To install bash just do: apk add --update bash

**Comment by Andr3as** [comment]
At home my container works as it should 🙈.

Now I was able to test the solution from above. I worked for me both on Alpine and on Debian. Could you please verify it on your system?
